docs/source/concepts/population.rst

```diff
@@ -9,8 +9,77 @@ Population Management
    :local:
    :backlinks: none
 
-Outline
--------
- - Initialization of simulants and the dependency problem.
- - Managing access with population views.
- -
+Since ``Vivarium`` is an agent-based simulation framework, managing a group of
+:term:`simulants` and their attributes is a critical task. Fundamentally, to run
+a simulation we need to be able to create new simulates, update their state
+attributes, and facilitate access to their state so that :term:`components` in
+the simulation can do interesting things based on it. The tooling to support
+working with our simulant population is called the population management system.
+
+The State Table
+---------------
+
```

Comment:
I think the best way to think about this document is to take it a section at a time.  I think talking about the simulation state is the right first thing to do.  We want to try to separate the talk about the intention/theory, and then illustrate the implementation with code examples.  A bunch of the images I made for the presentations might be useful here.  They're all at https://github.com/collijk/vivarium-demo/tree/master/2019_07_25_ihme/images

I would talk about the simulation state a bit abstractly at first.  E.g. it's the set of all information about the whole population *at a particular point in time*.  Here's a picture of a group of people.  Think about what attributes they might have (blood pressure, salary, ate a hamburger today).  

Then move on to the representation, organizing all the state info into rows and columns.

You can then use code examples from the examples sub-package to show the actual state table (doctest blocks will both display code and the outputs if I remember correctly).